### PR TITLE
Single table inheritance

### PR DESCRIPTION
**Pull request by mattKendon**
See https://github.com/jgrossi/corcel/pull/73 for details and discussion of this pull request

### CODE REVIEW

**Comment by jgrossi** [comment]
Please, paste the phpunit result here just to store that everything is ok with the tests :-)

File: tests/PostTest.php [hunk 1]
```diff
@@ -149,4 +149,23 @@ public function testCustomFieldWithAccessors()
         $this->assertEquals($post->title, 'Hello world!');
         $this->assertEquals($post->meta->title, 'New title');
     }
+
+    public function testSingleTableInheritance()
```
Comment (josec89) [inline tests/PostTest.php]: Docblock for these methods? :D

**Comment by josec89** [comment]
Great job with this! :) I will definitely use this for ACF and so many things..

:+1: :+1:

**Comment by mattKendon** [comment]
```Testing started at 14:45 ...
PHPUnit 4.2.2 by Sebastian Bergmann.

Configuration read from C:\xampp\htdocs\corcel\phpunit.xml



Time: 188 ms, Memory: 8.00Mb

OK (31 tests, 210 assertions)

Process finished with exit code 0```

**Comment by jgrossi** [comment]
Very nice job @mattKendon thanks again!

**Comment by mattKendon** [comment]
Happy to help, @jgrossi. I Found I needed this functionality and thought that others might as well.

**Comment by jgrossi** [comment]
Very nice! :-)

**Pull request merged**